src/atoms/feedCardItem/feedCardItem.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { useTheme } from 'styled-components';
+import { Theme } from '../../theme';
+import { LogoUrlBase } from '../../utils/constants';
+import {
+  ChevronDownIcon,
+  LinkIcon, Meatball, TwitterColor,
+} from '../icons';
+import { IconWrapper } from '../icons/iconWrapper';
+import { Text } from '../texts/text';
+import {
+  CenterContentContainer,
+  HoverItemsContainer,
+  MasterContainer,
+  TxTypeContainer,
+  TxTypeWrapper,
+  XPartyContent,
+  YPartyContent,
+  ChevronButton,
+  IconContainer,
+  SecondaryActionContainer,
+  ChainIcon,
+  DexIcon,
+} from './feedCardItem.styles';
+import { chainIcons, dexIcons } from './icons';
+import { FeedCardItemProps } from './types';
+
+export const FeedCardItem = (
+  {
+    isMulti,
+    txData,
+    handleToggle,
+    isOpen,
+    isFirst,
+  }: FeedCardItemProps,
+) => {
+  const theme = useTheme() as typeof Theme;
+  const [hover, setHover] = useState<boolean>(false);
+  const token0Usd = `($${txData.token0_amount_usd.toFixed(2)})`;
+  const token1Usd = `($${txData.token1_amount_usd.toFixed(2)})`;
+  // TODO: Add all tx types here
+  const txTypePreposition = txData.tx_type === 'swap' ? 'to' : 'text';
+  return (
+    <MasterContainer
+      onMouseEnter={() => setHover(true)}
+      onMouseLeave={() => setHover(false)}
+      key={Math.random()}
+      isMulti={isMulti}
+    >
+      <TxTypeWrapper>
+        <IconContainer>
+          <ChainIcon><IconWrapper icon={(chainIcons as any)[txData.chain]} height="14px" width="14px" /></ChainIcon>
```

Comment:
if we access the icon by `.chain` you are already assuming that it will match the key of chainIcons i think then its safe to simply give `.chain` the right type